cupy/_sorting/search.py

```diff
@@ -212,6 +212,26 @@ def where(condition, x=None, y=None):
     return _where_ufunc(condition.astype('?'), x, y)
 
 
+def argwhere(a):
+    """Return the indices of the elements that are non-zero.
+
+    Returns a (N, ndim) dimantional array containing the
+    indices of the non-zero elements. Where `N` is number of
+    non-zero elements and `ndim` is dimention of the given array.
+
+    Args:
+        a (cupy.ndarray): array
+
+    Returns:
+        cupy.array: Indices of elements that are non-zero.
+
+    .. seealso:: :func:`numpy.argwhere`
+
+    """
+    util.check_array(a, arg_name='a')
+    return a.argwhere()
```

Comment:
There's not a method for this one. Would just use the function `_ndarray_argwhere` directly.

```suggestion
    return _ndarray_argwhere(a)
```

<br>
<br>

Would need this `import` at the top of the file.

```python
from  cupy.core._routines_indexing import _ndarray_argwhere
````

Reply:
I have declared `cpdef ndarray argwhere(self)` inside `ndarray` in `core.pxd` there.

Reply:
and it works also when I run pytest.

Reply:
Yeah I think the point here is that `argwhere` is not a method on NumPy arrays. As CuPy likes to stay closely aligned with NumPy, they probably would prefer to not have this method. Maybe you could make it private if it is easier for you to implement. Though using the kernel directly seems like the safest bet 🙂

Reply:
Okay, I will make that change soon : )